app/src/main/java/com/github/sdpcoachme/EditProfileActivity.kt

```diff
@@ -44,12 +58,23 @@ class EditProfileActivity : ComponentActivity() {
  * Composable used to display the user's profile.
  */
 @Composable
-fun Profile() {
+fun Profile(email: String, database: Database, future: CompletableFuture<UserInfo>) {
```

Comment:
database doesn't need to be passed as a parameter, it can be retrieved by doing
```val context = LocalContext.current``` in the composable and then ```(context.applicationContext as CoachMeApplication).database``` to get the instance